solutions/예산.js

```diff
@@ -0,0 +1,42 @@
+const INPUT_PATH = "./inputs/예산.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(inputArguments) {
+  const [states, request ,budget] = inputArguments;
+
+  const requestList = request.split(" ").map(Number).sort((a, b) => a - b);
+
+  const requestSum = requestList.reduce((a, b) => a + b);
```

Comment:
```suggestion
  const [states, request, budget] = inputArguments;
  
  const requestList = request.split(" ").map(Number).sort((a, b) => a - b);
  const requestSum = requestList.reduce((a, b) => a + b);
```
인덴팅 맞춰주시면 좋을 것 같습니다!